src/pages/index-beta_t8apO3.astro

```diff
@@ -0,0 +1,50 @@
+---
```

Comment:
This is the new design, hence the weird URL so it's not really publicly accessible. Besides, it should be finalised in the near future, I think this is fine to leave in for now.